tailwind.purge.safelist.txt

```diff
@@ -38,3 +39,33 @@ tw-pb-8
 md:tw-pb-8
 lg:tw-pb-8
 xl:tw-pb-8
+
+# Themable color classes
+tw-bg-primary
```

Comment:
Hmm, now thinking we wouldn't ever set a `tw-bg-*` class within a contentful span.